resilience4j-ratelimiter/src/main/java/io/github/resilience4j/ratelimiter/internal/SemaphoreBasedRateLimiter.java

```diff
@@ -260,6 +263,13 @@ private void publishRateLimiterAcquisitionEvent(boolean permissionAcquired, int
         eventProcessor.consumeEvent(new RateLimiterOnFailureEvent(name, permits));
     }
 
+    @Override
+    public void close() throws IOException {
```

Comment:
Please, add a few words into documentation of this class on how and when the user needs to call this method.